### DISCUSSION TITLE
Light baking inside Bevy ?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [bevyengine/bevy#10724] [Q&A] [answered]
Hi,

Do you think that at some point, Bevy renderer will be able to bake the light for a given scene without needing the external help of Blender or other ?

Thanks!

**Comment by alice-i-cecile** [comment, marked as answer]
@pcwalton is working on something to this effect I believe :) Might be a bit before all the pieces land though.

**Reply by ThomasCartier** [reply]
Waw that would be great! 🙂

**Reply by JMS55** [reply]
To clarify, support for rendering _using_ light maps is likely to land in Bevy 0.13 as part of StandardMaterial. Support for _baking_ lightmaps within Bevy is unlikely to happen for a good while. We need some infrastructure first like an editor or CLI-based asset baker, and a better scene system (instead of treating GLTF files as scenes).

**Reply by ThomasCartier** [reply]
Ok I get it, so I will need to find a different way then
